integration/spark/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableDropHivePartitionCommand.scala

```diff
@@ -106,6 +108,32 @@ case class CarbonAlterTableDropHivePartitionCommand(
             ifExists,
             purge,
             retainData).run(sparkSession)
+          val isPartitionDataTrashEnabled = CarbonProperties.getInstance()
+            .getProperty(CarbonCommonConstants.CARBON_ENABLE_PARTITION_DATA_TRASH,
+              CarbonCommonConstants.CARBON_ENABLE_PARTITION_DATA_TRASH_DEFAULT).toBoolean
+          if (isPartitionDataTrashEnabled) {
+            // move  the partition files to trash folder which are dropped
+            val droppedPartitionNames = partitions.map { partition =>
+              partition.spec.map { specs => specs._1 + "=" + specs._2 }
+            }
+            val timeStamp = CarbonUpdateUtil.readCurrentTime();
```

Comment:
please directly use System.CurrentMillis, no need to call method for one line

Reply:
suppose if want to replace this System.CurrentMillis with some other in future then need to replace in all places. So i just used method call instead of directly using.

Reply:
for time and in this case it will be milliSec only and many places used directly, so use directly only

Reply:
ok